UnityComponent/DoomMaze/ProjectSettings/ProjectSettings.asset

```diff
@@ -259,7 +259,14 @@ PlayerSettings:
   androidGamepadSupportLevel: 0
   AndroidValidateAppBundleSize: 1
   AndroidAppBundleSizeToValidate: 150
-  m_BuildTargetIcons: []
+  m_BuildTargetIcons:
```

Comment:
This icon is really cool.  Btw, please add a source for the icon to the credits screen if it's not an original creation.

Reply:
done